<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopupDialog</class>
 <widget class="QDialog" name="PopupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>確認</string>
  </property>
  <widget class="QLabel" name="msgLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>キャンセル</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>確認</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
